src/Sarif/Core/WebMessageUtilities.cs

```diff
@@ -67,7 +69,8 @@
                 totalLength += length;
             }
 
-            totalLength += CRLF.Length;                 // Skip past the empty line;
+            totalLength += requestString.StartsWith(NewLineForWindows)
+                ? NewLineForWindows.Length : NewLineForNonWindows.Length; // Skip past the empty line;
```

Comment:
// Skip past whatever version of newline we found.
requestString.StartsWith(NewLineForWindows)
                ? NewLineForWindows.Length 
                 : NewLineForNonWindows.Length;

Reply:
fixed